packages/next/next-server/server/normalize-page-path.ts

```diff
@@ -8,6 +8,9 @@ export function normalizePagePath(page: string): string {
   if (!page.startsWith('/')) {
     page = `/${page}`
   }
+  if (page.endsWith('/')) {
```

Comment:
Does this need `page !== '/'` too?

Reply:
`/` can't reach this code as higher up there is 
```js
  if (page === '/') {
    page = '/index'
  }
```
but, I have the feeling that that piece is a bit out of place in this function. Removing it causes mayhem, but might be necessary in the future to get the `/index` behavior sorted out. 

To be on the safe side, in case it ever gets removed, I will use the same regex as in
```js
pathname = pathname.replace(/(?!^)\/$/, '')
```